### PR TITLE
feat: remove obsolete icons [breaking change]

### PR DESCRIPTION
**Pull request by kalilistic**
Remove obsolete icons. Potentially breaking change for the next API level.

### CODE REVIEW

File: Dalamud/Interface/FontAwesome/FontAwesomeIcon.cs [hunk 1]
```diff
@@ -3375,20 +2619,6 @@ public enum FontAwesomeIcon
     [FontAwesomeCategoriesAttribute(new[] { "Maps" })]
     Directions = 0xF5EB,
 
-    /// <summary>
-    /// The Font Awesome "discord" icon unicode character.
-    /// </summary>
-    [Obsolete]
-    [FontAwesomeSearchTerms(new[] { "discord" })]
-    Discord = 0xF392,
```
Comment (avafloww) [inline Dalamud/Interface/FontAwesome/FontAwesomeIcon.cs:3383]: Do we want to make an exception to keep the Discord icon, i.e. for Discord links within plugins?

**Review by avafloww — APPROVED** [review]
@kalilistic do we still want to do this? I'm ok with bringing it into v9 if so, just had a question about one icon.

File: Dalamud/Interface/FontAwesome/FontAwesomeIcon.cs (hunk 1 above)
Comment (kalilistic) [reply Dalamud/Interface/FontAwesome/FontAwesomeIcon.cs:3383]: None of these icons actually render today (including Discord). Font Awesome includes them in a separate "brands" file. This was excluded to keep the atlas manageable.

Comment (avafloww) [reply Dalamud/Interface/FontAwesome/FontAwesomeIcon.cs:3383]: Ah, got it!

**Pull request merged**